src/components/filters/MeasureValueFilter/DropdownBody.tsx

```diff
@@ -26,32 +27,36 @@ export interface IDropdownBodyOwnProps {
     locale?: string;
     disableAutofocus?: boolean;
     onCancel?: () => void;
-    onApply: (operator: string, value: IMeasureValueFilterValue) => void;
+    onApply: (operator: string, value: IMeasureValueFilterValue, enabledTreatNullValuesAs?: number) => void;
```

Comment:
The name of the new parameter is not matching its type

Reply:
Anyway, I would prefer to forget that it can be other than 0 somewhere in the API and use boolean everywhere it is possible.